app/modules/relationships/models.py

```diff
@@ -43,7 +43,7 @@ def delete(self):
         relationship.delete()
 
 
-class Relationship(db.Model, Timestamp):
+class Relationship(db.Model, HoustonModel, Timestamp):
```

Comment:
HoustonModel inherits FeatherModel inherits TimestampViewed inherits Timestamp so Timestams shouldn't be needed anymore